starwars/src/components/Cards.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import './card.css';
+
+ const Card = props => {
+  return (
+    <div className="card-component">
+      { props.starwarsChars.map(character => {
+          return (
+            <ProfileDescription 
+              name={character.name}
+              birthYear={character.birth_year}
+              gender={character.gender}
+              height={character.height}
+              mass={character.mass}
```

Comment:
remember what we discussed in zoom: consider passing the whole `character` object as opposed to its individual properties